ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -2062,7 +2062,7 @@
             "source": [
                 "# Args",
                 "\n",
-                "\nArgs are used to hold data related to the partition and additional information required to performe calculation at import and report. ",
+                "\nArgs are used to hold data related to Group Level and additional information required to performe calculation at import. ",
```

Comment:
Reset this change, it is probably a conflict not resolved correctly.

Reply:
Actually, this changed is wanted, The new Args (used for Yield Curve) does not have a Reporting Node and therefore a specific Partition. I used the definition Group to label the ensemble of every Partitions (for a give Period and Scenario)